File: src/main/java/org/labun/jooq/codegen/config/DatabaseConfig.java
package org.labun.jooq.codegen.config;

import lombok.Getter;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Konstantin Labun
 */
@Getter
@Setter
public class DatabaseConfig {
    private String username;
    private String password;
    private String driverClass;
    private String dbMetaClass;
    private String jdbcUrl;
    private List<String> schemas = new ArrayList<>();
}

File: src/main/kotlin/org/labun/jooq/generator/config/DatabaseConfig.kt
package org.labun.jooq.generator.config

/**
 * @author Konstantin Labun
 */
data class DatabaseConfig(
    var username: String = "",
    var password: String = "",
    var driverClass: String = "",
    var dbMetaClass: String = "",
    var jdbcUrl: String = "",
    var schemas: List<String> = emptyList()
)